doc/api/vm.md

```diff
@@ -437,6 +437,9 @@ changes:
     `cachedData` property of the returned `vm.Script` instance.
     The `cachedDataProduced` value will be set to either `true` or `false`
     depending on whether code cache data is produced successfully.
+  * `resolveDynamicImport` {Function} See [`module.link()`][]
+    * `specifier` {string}
+    * `scriptOrModule` {Script | Module}
```

Comment:
I wonder if the second argument here is necessary for these use cases if it will always be the top-level script? (I know we're going in circles around this somewhat, but helps to clarify!)

Reply:
i think it's quite likely most people won't assign a handler to each individual script or module instance so they will depend on it being inherited. I can leave a note explaining this behaviour if you think that would help people.

Reply:
Then perhaps just document it as `parentScript` as this format of the function is always a script? Alternatively might it make sense to pass this as a `this` binding?

Reply:
parentScriptOrModule works I guess, I'll change it in a bit

Reply:
Agreed.